geometry_msgs/PointStamped point
string motion_name
---
